src/psycopmlutils/data_checks/flattened/data_integrity.py

```diff
@@ -367,10 +367,11 @@ def run_validation_requiring_split_comparison(
             msg.warn(f"Failed checks: {failed_checks}")
 
 
-def check_feature_set_integrity_from_dir(  # noqa pylint: disable=too-many-statements
+def save_feature_set_integrity_from_dir(  # noqa pylint: disable=too-many-statements
```

Comment:
I think the data checking becomes a bit too implicit with this renaming. How about `check_and_save_feature_set_integrity_from_dir`?

Reply:
Hmm, I tend to reserve "check" functions to ones that either raise an error or return something if checks don't pass. Whereas here, all we get is the output in an .html, right?

Open to changes 👍